safe-global/safe-wallet-web
# feat: [Counterfactual] Update UI elements

## What it solves

Resolves #3207

## How this PR fixes it

- Updates the overview widget design
- Adds a "First steps" widget that is only visible for undeployed safes
- Hide every other widget for undeployed safes
- Hide the overview buttons for undeployed safes
- Shows an info message on the assets page with the option to activate the safe immediately
- Displays a new First transaction dialog
- Shows an indicator in the sidebar about the safe account status

## How to test it

1. Only the First steps widget should be visible
2. There should be no buttons visible in the overview widget
3. There should be an info message on the assets page with an "Activate now" button
4. Pressing "Create transaction" should display the new dialog with multiple options
5. There should be a warning indicator in the sidebar as long as the safe is not deploying
6. Once a deployment is underway the indicator should turn blue and spin

## Screenshots
<img width="1270" alt="Screenshot 2024-02-15 at 15 15 15" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/191e0e2c-fa76-4973-b02f-53aa4d974051">
<img width="648" alt="Screenshot 2024-02-14 at 16 44 21" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/f671c03e-3d64-4731-a114-cafeb70bf6f6">

<img width="644" alt="Screenshot 2024-02-14 at 16 42 26" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/9f7abfb2-2c57-4d93-b38c-29cfe26971fd">
<img width="696" alt="Screenshot 2024-02-14 at 16 55 53" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/14d918c3-6f83-46d4-b4b6-fd63812de04e">


## Checklist
